frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -53,29 +45,31 @@ function LandingPage() {
         <title>Home - Health Equity Tracker</title>
         <link rel='preload' as='image' href='/img/stock/family-laughing.png' />
       </Helmet>
+
       <h2 className='sr-only'>Home Page</h2>
-      <Grid container className={styles.Grid}>
-        <Grid
-          container
-          className={styles.HeaderRow}
-          direction='row'
-          justifyContent='center'
-          alignItems='center'
-        >
-          <Grid item className={styles.HeaderTextItem} xs={12} sm={12} md={7}>
-            <Typography
+      <div className='m-auto flex w-full max-w-newsPage flex-wrap'>
+        <div className='flex flex-wrap items-center justify-center border-0 border-b border-solid pb-8 pt-4'>
+          <div className='w-full px-12 py-4 md:w-7/12'>
+            <h3
               id='main'
-              className={styles.HeaderText}
-              variant={pageIsWide ? 'h3' : 'h4'}
-              paragraph={true}
-              component='h3'
+              className={`
+              mb-4 mt-0
+              pb-4 pt-12
+              ${
```

Comment:
Can we use the tailwind break point modifiers instead? Like "text-color xl:text-other-color" to set the color based on screen width